### PR TITLE
Node Coloring MVC move

### PR DESCRIPTION
**Pull request by mihirsamdarshi**
Also contains fix for locking sliders not working

### CODE REVIEW

**Comment by mihirsamdarshi** [comment]
this PR contains code clean up, the fix for the lock sliders option, and beginning stages of a node coloring move which has been disabled/commented out

**Comment by mihirsamdarshi** [comment]
ready for review @dondi

File: web-client/public/js/update-app.js [hunk 1]
```diff
@@ -71,7 +77,7 @@ const displayNetwork = (network, name) => {
     $("input[type='range']").off("input"); // I have no idea why I do this. Investigate later.
 
     // If more things need to be turned off, we'll add them to this array
-    [ "#resetSliders", "#resetSlidersMenu", "#undoReset", "#undoResetMenu" ].forEach(
+    [ "#resetSliders", RESET_SLIDERS_MENU_OPTION, "#undoReset", UNDO_SLIDER_RESET_MENU ].forEach(
```
Comment (dondi) [inline web-client/public/js/update-app.js]: The conversion to constants can be useful but is half-done here. May as well convert `#resetSliders` and `#undoReset` here too.

File: web-client/public/js/update-app.js [hunk 2]
```diff
@@ -345,5 +348,22 @@ export const updateApp = grnState => {
...
+        $(AVG_REPLICATE_VALS_TOP_MENU + " span").removeClass("glyphicon-ok");
+        $(AVG_REPLICATE_VALS_TOP_MENU).removeProp("checked");
+        $(AVG_REPLICATE_VALS_TOP_SIDEBAR).removeProp("checked");
+        updaters.renderNodeColoring();
+    }
+
     refreshApp();
+
+*/
+
```
Comment (dondi) [inline web-client/public/js/update-app.js]: Maybe plan to eliminate comments soon after merging? I know they have some use while in the middle of things, but beyond that, let GitHub do the change-tracking instead.

**Pull request merged**